src/PProf.jl

```diff
@@ -217,8 +217,35 @@ function pprof(data::Union{Nothing, Vector{UInt}} = nothing,
         writeproto(io, prof)
     end
 
-    if web run(`$go_pprof -http=$webhost:$webport $out`) end
+    if web
+        if !isnothing(PProf.pprof_proc[1])
+            kill(PProf.pprof_proc[1])
+        end
+
+        PProf.pprof_proc[1] = run(`$go_pprof -http=$webhost:$webport $out`, wait = false)
+    end
```

Comment:
I think it would be nice to move this into a function, something like "refresh_pprof()", so that if the user updates the output file manually, they can refresh the server without having to rerun `pprof()`, which would overwrite the file.